File: pgm/ABC_Java/ABC_319_A.java
import java.io.*;
// import java.math.*;
import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String S = reader.readLine();
        Map<String, Integer> m = new HashMap<>();
        m.put ("tourist", 3858);
        m.put ("ksun48", 3679);
        m.put ("Benq", 3658);
        m.put ("Um_nik", 3648);
        m.put ("apiad", 3638);
        m.put ("Stonefeang", 3630);
        m.put ("ecnerwala", 3613);
        m.put ("mnbvmar", 3555);
        m.put ("newbiedmy", 3516);
        m.put ("semiexp", 3481);
        System.out.println (m.get (S));
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_319_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.*
// import java.util.StringTokenizer

// import kotlin.math.*

fun main() {
    val S: String = readln()
    val m: Map<String, Int> = mapOf(
        "tourist" to 3858,
        "ksun48" to 3679,
        "Benq" to 3658,
        "Um_nik" to 3648,
        "apiad" to 3638,
        "Stonefeang" to 3630,
        "ecnerwala" to 3613,
        "mnbvmar" to 3555,
        "newbiedmy" to 3516,
        "semiexp" to 3481
    )
    println(m[S])
    kotlin.system.exitProcess(0)
}